xml/System.Runtime.InteropServices.CustomMarshalers/TypeToTypeInfoMarshaler.xml

```diff
@@ -36,11 +36,11 @@
   </Docs>
   <Members>
     <Member MemberName=".cctor">
-      <MemberSignature Language="C#" Value="public TypeToTypeInfoMarshaler ();" />
+      <MemberSignature Language="C#" Value="public static TypeToTypeInfoMarshaler ();" />
```

Comment:
Is it really a public static constructor? I don't have .NET 1.1 (API browser claims that it's a latest version where this member is available), but in modern environment one cannot declare static constructor as public in C#, it generates compiler error CS0515. 

Is there even a reason why API Docs should include static constructor? It is not really an "API", in a sence that one cannot use it directly. Moreover, currently the docs do not seem to provide any useful information about it, besides the fact it was removed in .NET 2.0.

Reply:
You're correct. It's a static constructor for a public class. I've removed the public access modifier for the C# and VB.Net static constructor. C++ does not support static constructors, so I've reverted the addition of the static modifier.

While a static constructor cannot be accessed directly, it is called prior to any first instance/usage of the type, which can lead to unusual exceptions such as [TypeInitializationException](https://docs.microsoft.com/en-us/dotnet/api/system.typeinitializationexception?view=netframework-4.8#remarks) which explicitly note "_What often makes this exception difficult to troubleshoot is that static constructors are not always explicitly defined in source code._" As such, the presence of a static constructor should be noted in the documentation.

A deprecated static constructor is listed for each of the 4 classes in the [System.Runtime.InteropServices.CustomMarshalers Namespace](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.custommarshalers?view=netframework-1.1). A separate page for the static constructors is probably unnecessary, but their presence should be noted as a remark.

Appreciate the feedback!

Reply:
Thanks for submitting this PR and raising the issue of a static constructor appearing in our doc set, @EugeneKramer. Since method signatures are gathered using reflection, I'm not sure that the changed signature won't be overwritten by the next reflection. But I'm also not sure why a static constructor is documented in the first place -- though I don't recall whether 2.0 introduced a change in the way static constructors were handled from 1.0/1.1. 

@dend @joelmartinez, do you have any sense of why this constructor is marked as public and appears in the documentation? My guess would be that this is the only static constructor documented in the managed reference.,

I would think that the ILAsm signature also needs to change, although I no longer have any .NET Framework 1.1 assemblies.

 For the moment, I'll mark this PR as blocked.

Reply:
MSDN also had this constructor:
https://msdn.microsoft.com/en-us/library/03faz537(v=vs.80).aspx?toc=xxx

with the same signature that @EugeneKraner is proposing. 

@rpetrusha you can access any version of the assemblies at https://apidrop.visualstudio.com/_git/binaries?path=%2Fdotnet&version=GBmaster

Reply:
apisof.net also lists this constructor, so I think it's fine to have it if we fix the signature
https://apisof.net/catalog/System.Runtime.InteropServices.CustomMarshalers.TypeToTypeInfoMarshaler..cctor()

Reply:
I still find this bizarre, but the static constructor is clearly public in .NET Framework 1.1. So correcting the syntax is appropriate. Is the correct syntax likely to be overwritten, @mairaw?

Reply:
> Is the correct syntax likely to be overwritten

Yes ... this needs to be fixed in mdoc's signature formatters. It will reset these next time an update is run.

Reply:
@rpetrusha - As mentioned earlier, there are public static constructors listed for each of the 4 classes in the `System.Runtime.InteropServices.CustomMarshalers Namespace` with the same public signature. If this PR is going through, should each of their signatures be updated...or should a separate PR be opened?

[System.Runtime.InteropServices.CustomMarshalers Namespace](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.custommarshalers?view=netframework-1.1):

> Provides the static class constructor.

1. [EnumerableToDispatchMarshaler Constructor](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.custommarshalers.enumerabletodispatchmarshaler.-cctor?view=netframework-1.1)
1. [EnumeratorToEnumVariantMarshaler Constructor](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.custommarshalers.enumeratortoenumvariantmarshaler.-cctor?view=netframework-1.1)
1. [ExpandoToDispatchExMarshaler Constructor](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.custommarshalers.expandotodispatchexmarshaler.-cctor?view=netframework-1.1)
1. [TypeToTypeInfoMarshaler Constructor](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.custommarshalers.typetotypeinfomarshaler.-cctor?view=netframework-1.1)

All were deprecated in .Net 2.0.

Reply:
@joelmartinez I've opened an internal mdoc bug for this:
[Bug 97094: [mdoc] wrong signature for public static constructors in .NET Framework 1.1](https://ceapex.visualstudio.com/Engineering/_queries/edit/97094)